platform.js

```diff
@@ -154,6 +154,19 @@ class App {
 
     const dest = path.join(this.originalResourcesDir, 'app.asar')
     debug(`Running asar with the options ${JSON.stringify(asarOptions)}`)
+
+    if (asarOptions.filename) {
```

Comment:
I think that if there are other `asarOptions` other than `filename` and `filename` is set, it should warn that the other options won't be used.

Reply:
should that happen here? or in the argument parsing?

Reply:
It should happen here. If it happens in argument parsing, people who use the JS API (such as users of Electron Forge or ember-electron) will not get the warning.